src/styles/utils/extends.scss

```diff
@@ -0,0 +1,104 @@
+%playfair {
+  font-family: "Playfair Display", serif;
+  font-style: normal;
+}
+
+%raleway {
+  font-family: "Raleway", sans-serif;
+  font-style: normal;
+}
+
+%container {
+  box-sizing: border-box;
```

Comment:
box-sizing: border-box should be applied to everything, like * { box-sizing: border-box }